connectors/webhook/src/main/java/io/camunda/connector/inbound/model/WebhookConnectorProperties.java

```diff
@@ -111,6 +110,15 @@ public record WebhookConnectorProperties(
         @FEEL
         HMACScope[] hmacScopes,
     WebhookAuthorization auth,
+    @TemplateProperty(
+            id = "responseExpression",
+            label = "Response expression",
+            type = PropertyType.Text,
```

Comment:
When should we use Text and when should we use String, out of curiosity?

Reply:
I think there is a difference regarding rendering but I would have to look it up.

Reply:
https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#types